tools/manual_testing/plot.ipynb

```diff
@@ -0,0 +1,571 @@
+    "        'yy':sc.Variable(dims=['event'], values=np.random.rand(N)),\n",
+    "        'zz':sc.Variable(dims=['event'], values=np.random.rand(N))\n",
+    "    })\n",
+    "binned = sc.bin(data, edges=[sc.linspace(dim='xx', start=0.0,stop=1.0,num=10),\n",
+    "                             sc.linspace(dim='yy', start=0.0,stop=1.0,num=4),\n",
+    "                             sc.linspace(dim='zz', start=0.0,stop=1.0,num=4)])\n",
+    "binned.plot(resolution=100)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a3c6f435-5f01-4923-85c7-8fd4195cbec0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binned['xx', 0].plot(resolution=100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d4bc382-f2ac-45ff-a447-53fc0ffe0f38",
+   "metadata": {},
+   "source": [
+    "Known issues (to fix):\n",
+    "- Slider steps are based on coarse underlying binning\n",
+    "- Profile selection based on coarse underlying binning"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4785c953-ff34-4496-996e-fa931df16ee3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binned['xx', 0].plot(projection='1d', resolution=100)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab160e28-4b5c-4f61-8c15-ae0af7ee7892",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binned['xx', 0]['yy', 0].plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f1382619-4769-4683-b46f-70bb54f20ecf",
+   "metadata": {},
+   "source": [
+    "# 3d plotting\n",
+    "Known issues (to fix):\n",
+    "- Profile button should be disabled or not shown, until supported\n",
+    "- Clicking buttons to switch dimensions don't work and raise exception"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "525d4ea1-7fa3-469b-807d-cfc1b4b4f4a0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipp as sc\n",
+    "import numpy as np\n",
+    "N = 10\n",
+    "M = 40\n",
+    "L = 30\n",
+    "K = 20\n",
+    "xx = np.arange(3, N, dtype=np.float64)\n",
+    "yy = np.arange(4, M, dtype=np.float64)\n",
+    "zz = np.arange(5, L, dtype=np.float64)\n",
+    "qq = np.arange(6, K, dtype=np.float64)\n",
+    "x, y, z, q = np.meshgrid(xx, yy, zz, qq, indexing='ij')\n",
+    "b = N/20.0\n",
+    "c = M/2.0\n",
+    "d = L/2.0\n",
+    "r = np.sqrt(((x-c)/b)**2 + ((y-c)/b)**2 + ((z-d)/b)**2  + ((q-d)/b)**2)\n",
+    "a = np.sin(r)\n",
+    "d = sc.Dataset()\n",
+    "da = sc.DataArray(data=sc.Variable(dims=['xx', 'yy', 'zz', 'Q_x'], values=a, unit='K'))\n",
+    "da.coords['xx'] = sc.Variable(dims=['xx'], values=xx)\n",
+    "da.coords['yy'] = sc.Variable(dims=['yy'], values=yy, unit='m')\n",
+    "da.coords['zz'] = sc.Variable(dims=['zz'], values=zz)\n",
+    "da.coords['Q_x'] = sc.Variable(dims=['Q_x'], values=qq)\n",
+    "sc.plot(da, projection='3d', aspect='auto')"
+   ]
```

Comment:
The outline box used to reach just on the edge of the pixels outer, instead of in the center as is now, which makes the ticks less readable.
![Screenshot at 2021-07-09 11-25-04](https://user-images.githubusercontent.com/39047984/125056687-c4740f00-e0a8-11eb-8c5c-b3c1aaf46b0a.png)